docs/java-rest/high-level/dataframe/get_data_frame.asciidoc

```diff
@@ -0,0 +1,45 @@
+--
+:api: get-data-frame-transform
+:request: GetDataFrameTransformRequest
+:response: GetDataFrameTransformResponse
+--
+[id="{upid}-{api}"]
+=== Get Data Frame Transform API
+
+The Get Data Frame Transform API is used get one or more {dataframe-transform}.
+The API accepts a +{request}+ object and returns a +{response}+.
+
+[id="{upid}-{api}-request"]
+==== Get Data Frame Request
+
+A +{request}+ requires a data frame transform id or the special wildcard `_all`
```

Comment:
This endpoint also supports a comma delimited list of transform IDs, similar to ML GET Jobs.